CondCore/CondDB/test/BuildFile.xml

```diff
@@ -17,11 +18,9 @@
 </bin>
 
 <bin   file="testRootStreaming.cpp" name="testRootStreaming">
-  <lib   name="testCondDBDict"/>
```

Comment:
I think root is going to take care of it by reading the testCondDBDict_xr.rootmap file ...right? at least unit tests runs locally without explicitly linking to testCondDBDict